### PR TITLE
Create some PSP folders when initializing emulator

### PR DESCRIPTION
**Pull request by FollowerOfBigboss**
Since firmware doesn't create folders for me i decided to create them all. I used them while i am testing features of vsh. It's not a big deal but makes my life easier.

Used resources:
https://playstationdev.wiki/pspdevwiki/index.php/File_Structures
https://playstationdev.wiki/pspdevwiki/index.php/Filesystem

### CODE REVIEW

**Comment by DragonNeos** [comment]
@FollowerOfBigboss On Line 152, shouldn't License be spelled with an S instead of the second C as referenced from your hyperlinks?

**Comment by FollowerOfBigboss** [comment]
@DragonNeos thank you for correcting me.

File: src/jpcsp/MainGUI.java [hunk 1]
```diff
@@ -136,8 +136,28 @@
     private List<RecentElement> recentUMD = new LinkedList<RecentElement>();
     private List<RecentElement> recentFile = new LinkedList<RecentElement>();
     private final static String[] userDir = {
+        "MP_ROOT/100ANV01",
+        "MP_ROOT/100MNV01",
```
Comment (gid15) [inline src/jpcsp/MainGUI.java:140]: MP_ROOT should be under ms0, right?

Comment (FollowerOfBigboss) [reply src/jpcsp/MainGUI.java:140]: @gid15 You are right, my brain was confused.

**Pull request merged**